src/cli/repo/filesystem.go

```diff
@@ -0,0 +1,215 @@
+package repo
+
+import (
+	"github.com/alcionai/clues"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/alcionai/corso/src/cli/config"
+	"github.com/alcionai/corso/src/cli/flags"
+	. "github.com/alcionai/corso/src/cli/print"
+	"github.com/alcionai/corso/src/cli/utils"
+	"github.com/alcionai/corso/src/internal/events"
+	ctrlRepo "github.com/alcionai/corso/src/pkg/control/repository"
+	"github.com/alcionai/corso/src/pkg/repository"
+	"github.com/alcionai/corso/src/pkg/storage"
+)
+
+const (
+	fsProviderCmd          = "filesystem"
```

Comment:
Nit: long form `Command` for consts, short for `Cmd` for funcs, please.  It's not much, but it keeps things consistent.
```suggestion
	fsProviderCommand          = "filesystem"
```